lib/ansible/module_utils/redfish_utils.py

```diff
@@ -275,15 +271,24 @@ def clear_logs(self):
                         return response
         return {'ret': True}
 
-    def get_storage_controller_inventory(self):
+    def aggregate(self, func):
+        ret = True
+        entries = []
+        for systems_uri in self.systems_uris:
+            inventory = func(systems_uri)
+            ret = inventory.pop('ret') and ret
+            entries.append(inventory['entries'])
```

Comment:
Since we are now aggregating inventory info from multiple systems, the output should show which system the inventory info belongs to. Suggest changing L280 above to:

```
            entries.append(({'systems_uri': systems_uri}, inventory['entries']))
```

Reply:
thank you @billdodd :)